composer.json

```diff
@@ -40,7 +40,7 @@
     "OSL-3.0"
   ],
   "require": {
-    "nosto/php-sdk": "5.0.*",
+    "nosto/php-sdk": "^5.1.0",
```

Comment:
Do we want fixed or 5.1.* would do it?

Reply:
Nopes, as long as we follow semantic versioning 5.* should be fine